protocol/dataframe_protocol.py

```diff
@@ -58,6 +58,9 @@ class DtypeKind(enum.IntEnum):
     CATEGORICAL = 23
 
 
+Dtype = Tuple[DtypeKind, int, str, str]
```

Comment:
```suggestion
Dtype = Tuple[DtypeKind, int, str, str]  # see Column.dtype
```

Could also maybe move the `Column.dtype` description here as evidently the dtype tuple gets returned elsewhere, but eh probably nicer to keep things as-is.

Reply:
Yeah, I wanted to put the `Column.dtype` docs closer to the type alias definition but it is a little awkward. The comment that you suggest is good compromise.